### PR TITLE
Update onboard-windows-server.md

### PR DESCRIPTION
**Pull request by msbemba**
Added License requirements to the top of the page 

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9467

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/onboard-windows-server.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ This topic describes how to onboard specific Windows servers to Microsoft Defend
 
 For guidance on how to download and use Windows Security Baselines for Windows servers, see [Windows Security Baselines.](/windows/security/threat-protection/windows-security-configuration-framework/windows-security-baselines)
 
+## License Requirement
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-windows-server.md]:
```suggestion
## License requirement
```

File: microsoft-365/security/defender-endpoint/onboard-windows-server.md [hunk 2]
```diff
@@ -41,6 +41,10 @@ This topic describes how to onboard specific Windows servers to Microsoft Defend
...
+
+In order to be eligible to purchase Microsoft Defender for Endpoint Server SKU, you must have already purchased a combined minimum of any of the following, Windows E5/A5, Microsoft 365 E5/A5 or Microsoft 365 E5 Security subscription licenses. For more information on licensing, see the [Product Terms](https://www.microsoft.com/licensing/terms/productoffering/MicrosoftDefenderforEndpointServer/all).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-windows-server.md]:
```suggestion
In order to be eligible to purchase Microsoft Defender for Endpoint Server SKU, you must have already purchased a combined minimum of any of the following: Windows E5/A5, Microsoft 365 E5/A5, or Microsoft 365 E5 Security subscription licenses. For more information on licensing, see the [Product Terms](https://www.microsoft.com/licensing/terms/productoffering/MicrosoftDefenderforEndpointServer/all).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**